service_crategen/src/main.rs

```diff
@@ -261,6 +261,18 @@ See [LICENSE][license] for details.
 //!
 //! If you're using the service, you're probably looking for [{client_name}](struct.{client_name}.html) and [{trait_name}](trait.{trait_name}.html).
 
+// =================================================================
```

Comment:
I think the warning should be at the top of the file, even before the module doc comment. It doesn't interfere with doc generation.